src/bisect.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+async function bisect (compare, getByIndex, length) {
+  let top = length
+  let bottom = 0
+
+  async function _bisect () {
+    const middle = Math.floor((top + bottom) / 2)
+    const data = await getByIndex(middle)
+    const cmp = compare(data)
+
+    if (cmp < 0) {
+      bottom = middle
+    } else {
+      top = middle
+    }
+
+    const newMiddle = Math.floor((top + bottom) / 2)
+    if (newMiddle === middle) {
+      return middle
```

Comment:
I think we should add opts.returnClosest option here as well
https://github.com/bitfinexcom/hypercore-bisect/blob/master/index.js#L49

Reply:
In our case, returnClosest should always be true, so it's not needed here